packages/neuron-wallet/src/models/keys/hd-public-key-info.ts

```diff
@@ -0,0 +1,49 @@
+import { AddressType } from "./address"
+import { AccountExtendedPublicKey } from "./key"
+
+export default class HdPublicKeyInfoModel {
+  public walletId: string
+  public address: string
+  public addressType: AddressType
+  public addressIndex: number
+  public publicKeyInBlake160: string
+  public description?: string
+
+  public get path(): string {
+    return `${AccountExtendedPublicKey.ckbAccountPath}/${this.addressType}/${this.addressIndex}`
```

Comment:
This function may be more expressively by implementing with `Address.pathFor()`.

Reply:
Reall good spot here 👍